### ISSUE TITLE
Fetch from PDB-REDO fails on first attempt

### ISSUE THREAD

**Issue by olibclarke** [pemsley/coot#87]
Hi Paul, using commit `a311017` on Mac, fetching from PDB-REDO fails if attempted immediately upon launch, with this error:

```
:::::::::::::::::::::: mode_name "pdb-redo"
PDB Accession Code: 3ch3
frame: 0x15134f070
DEBUG:: extracted accession code handle mode n 112
debug:: safe_python_command() PyRun_SimpleString() import get_ebi ; get_ebi.get_pdb_redo("3ch3")
DEBUG:: getting https://pdb-redo.eu/db/3ch3/3ch3_final.pdb
DEBUG:: in coot_get_url_and_activate_curl_hook https://pdb-redo.eu/db/3ch3/3ch3_final.pdb 3ch3_final.pdb
Failed to get https://pdb-redo.eu/db/3ch3/3ch3_final.pdb 3ch3_final.pdb status None
DEBUG:: getting https://pdb-redo.eu/db/3ch3/3ch3_final.mtz
DEBUG:: in coot_get_url_and_activate_curl_hook https://pdb-redo.eu/db/3ch3/3ch3_final.mtz 3ch3_final.mtz
Failed to get https://pdb-redo.eu/db/3ch3/3ch3_final.mtz 3ch3_final.mtz status None
DEBUG:: getting https://pdb-redo.eu/db/3ch3/3ch3_final.py
DEBUG:: in coot_get_url_and_activate_curl_hook https://pdb-redo.eu/db/3ch3/3ch3_final.py 3ch3.py
Failed to get https://pdb-redo.eu/db/3ch3/3ch3_final.py 3ch3.py status None
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/coot/get_ebi.py", line 384, in get_pdb_redo
    status_imol = read_pdb(pdb_file_name)
                  ^^^^^^^^
NameError: name 'read_pdb' is not defined
```

If I attempt the same operation using "Fetch PDB & map from EDS" it works fine; if I then reattempt the fetching from PDB-REDO with the same PDB-ID it succeeds, but this is presumably because the structure has been cached, as if I try it with a "fresh" PDB ID it fails with the same error.

Cheers
Oli

**Comment by pemsley** [comment]
Thanks - fixed in 898a79617a98c0b7f197c13d8c15d1b17e1166a9

**Comment by olibclarke** [comment]
Hi @pemsley,

This seems to work now, although I do notice it still generates an error in the log:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/coot/get_ebi.py", line 394, in get_pdb_redo
    exec(compile(open(py_file_name, "rb").read(), py_file_name, 'exec'))
  File "3ch3.py", line 3, in <module>
    interesting_things_gui ("PDB_REDO results",[
    ^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'interesting_things_gui' is not defined
```
